### PR TITLE
Initial model card and schema routes

### PR DESCRIPTION
**Pull request by a3957273**
This adds routes for:

- Get a model card (either the latest, or a specific version).
- Initialize a blank model card from a schema.
- Update a model card.

### CODE REVIEW

File: backend/src/models/v2/Model.ts [hunk 1]
```diff
@@ -13,6 +13,14 @@ export interface CollaboratorEntry {
   roles: Array<'owner' | 'contributor' | 'consumer' | string>
 }
 
+export interface ModelCardInterface {
+  schemaId: string
+  version: number
+  createdBy: string
```
Comment (ARADDCC002) [inline backend/src/models/v2/Model.ts:19]: Is this going to be just the username or is it worth adding in the full entity object?

Comment (a3957273) [reply backend/src/models/v2/Model.ts:19]: We should always use `users` whenever it's an action done by someone.  There's never a 'group' that creates a release, it's always a specific person.  (Also, entities are now strings of the form `${type}:${id}`).

File: backend/src/routes/v2/model/modelcard/getModelCard.ts [hunk 2]
```diff
@@ -25,20 +27,25 @@ interface GetModelCardResponse {
...
-      modelCard: {
-        modelId: 'example-model',
-        schemaId: 'example-schema',
+    let modelCard: ModelCardInterface
+    if (version === GetModelCardVersionOptions.Latest) {
+      const card = (await getModelById(req.user, modelId)).card
 
-        version: 1,
-        metadata: { example: true },
+      console.log('CARD', card)
```
[the remark this answers is not in the document]
Comment (a3957273) [reply backend/src/routes/v2/model/modelcard/getModelCard.ts:38]: My bad.  I had some issues mocking.  Nice catch!

[the remark this answers is not in the document]
Comment (ARADDCC002) [reply backend/src/routes/v2/model/modelcard/getModelCard.ts]: I do it all the time :D

File: backend/src/models/v2/Model.ts (hunk 1 above)
Comment (ARADDCC002) [reply backend/src/models/v2/Model.ts:19]: Yes sorry I forgot the change in entity format, yeah I agree with this in that case.

**Review by ARADDCC002 — APPROVED** [review]

File: backend/src/routes/v2/model/modelcard/getModelCard.ts [hunk 3]
```diff
@@ -25,20 +27,23 @@ interface GetModelCardResponse {
...
-      modelCard: {
-        modelId: 'example-model',
-        schemaId: 'example-schema',
+    let modelCard: ModelCardInterface
+    if (version === GetModelCardVersionOptions.Latest) {
+      const card = (await getModelById(req.user, modelId)).card
 
-        version: 1,
-        metadata: { example: true },
+      if (!card) {
```
Comment (JR40159) [inline backend/src/routes/v2/model/modelcard/getModelCard.ts:38]: Why is this logic not in the service?

**Pull request merged**